server/src/test/java/io/crate/integrationtests/DDLIntegrationTest.java

```diff
@@ -886,8 +886,8 @@ public void testAddGeneratedColumnToTableWithExistingGeneratedColumns() throws E
                                  "{\"dynamic\":\"strict\"," +
                                  "\"_meta\":{" +
                                  "\"generated_columns\":{" +
-                                 "\"added\":\"date_trunc('day', ts)\"," +
-                                 "\"day\":\"date_trunc('day', ts)\"}}," +
+                                 "\"day\":\"date_trunc('day', ts)\"," +
```

Comment:
With old behavior new column appeared first which somehow contradicts with `LinkedHashMap` usage, I guess it's because we added all at once with mirroring.